content/swift/concepts/sets/terms/contains.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '.contains()'
+Description: 'Check whether an item exists within the set.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`contains()`** method returns true if the given element exists in the set, false otherwise.
+
+## Syntax
+
+```pseudo
+if mySet.contains(number) {
+    print("This set contains this number: \(number)")
+} else {
+    print("This set does not contain this number: \(number)")
+}
```

Comment:
This if/else statement is good to use in the example, but can we stick to the bare-bones for the syntax portion? Also, let's name `mySet` -> `setName`

I was bouncing back between `setName` and `setInstance`, but the former naming may be less confusing. I'd love to know what you think as well